ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -26,7 +26,7 @@ func UpCommand() cli.Command {
 		Usage:  "Creates the ECS cluster (if it does not already exist) and the AWS resources required to set up the cluster.",
 		Before: ecscli.BeforeApp,
 		Action: cluster.ClusterUp,
-		Flags:  append(clusterUpFlags(), command.OptionalClusterFlag(), command.OptionalRegionFlag()),
+		Flags:  append(append(clusterUpFlags(), command.OptionalRegionFlag()), command.OptionalClusterAndProfileFlags()...),
```

Comment:
Go doesn't seem to provide a good way to concatenate multiple slices, so I see why we're doing this but it seems a little clunky. Would it be worth adding a helper to `flags.go`, say, `OptionalFlags` that concatenated `OptionalClusterAndProfileFlags` with `OptionalRegionalFlag` so that this line would just read as `append(clusterUpFlags(), command.OptionalFlags()...`? 

I realize that if we add more optional flags, we'd still run into the same issue of trying to concatenate multiple slices so maybe this is just a temporary hack, but just a thought, for improving readability at the top level as much as possible.